daisy/disk.go

```diff
@@ -66,6 +67,13 @@ type Disk struct {
 	compute.Disk
 	Resource
 
+	// If this is enabled, then WINDOWS will be added to the
+	// disk's guestOsFeatures. This is a string since daisy
+	// replaces variables after JSON has been parsed.
+	// (If it were boolean, the JSON marshaller throws
+	// an error when it sees something like `${is_windows}`)
+	IsWindows string `json:"isWindows,omitempty"`
```

Comment:
I believe you can do this:
IsWindows bool                `json:"isWindows,string,omitempty"`

Reply:
It convert the string "true|false" to bool

Reply:
The problem is that daisy populates variables *after* the JSON document has been parsed into a struct. So, the JSON parser will throw something like: `trying to unmarshal "${is_windows}" into bool`